ci/do_ci.sh

```diff
@@ -34,20 +36,22 @@ if [ -n "$CIRCLECI" ]; then
         mv "${HOME:-/root}/.gitconfig" "${HOME:-/root}/.gitconfig_save"
         echo 1
     fi
-    export BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} --jobs 8"
-    export BAZEL_TEST_OPTIONS="${BAZEL_TEST_OPTIONS} --jobs 8 --local_test_jobs=8"
-    export MAKEFLAGS="-j 8"
 fi
 
 if [ "$1" == "coverage" ]; then
     export CC=gcc
     export CXX=g++
+    CONCURRENCY=6
 else
     export PATH=/usr/lib/llvm-7/bin:$PATH
     export CC=clang
     export CXX=clang++
 fi
 
+export BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} --jobs ${CONCURRENCY}"
+export BAZEL_TEST_OPTIONS="${BAZEL_TEST_OPTIONS} --jobs ${CONCURRENCY} --local_test_jobs=${CONCURRENCY}"
+export MAKEFLAGS="-j ${CONCURRENCY}"
```

Comment:
Unrelated to this PR, but how come we have `MAKEFLAGS` here?

Reply:
that was me killing the CI concurrency/OOM issue with more fire then needed I guess, that doesn't make sense. Removed.